### PR TITLE
[FIX] web: legacy: customFilterMenu is focusable

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
@stefanorigano Hey, could you check the style ? I think the recent re-factor of dropdown style introduced some inconsistencies during focus

**Comment by stefanorigano** [comment]
Hello @kebeclibre ,

yes, indeed it solves the `:focus` issue, but the button it's still not "clickable" pressing [ENTER], only pressing [→].
Do you know how to solve it?

File: addons/web/static/src/legacy/scss/dropdown.scss [hunk 1]
```diff
@@ -54,3 +54,11 @@
         }
     }
 }
+
+.o_dropdown_menu {
```
Comment (stefanorigano) [inline addons/web/static/src/legacy/scss/dropdown.scss:58]: I think this code should be here ;)
https://github.com/odoo/odoo/blob/master/addons/web/static/src/core/dropdown/dropdown.scss#L24

**Comment by kebeclibre** [comment]
@stefanorigano The span cuts off all the features of a clickable element

from: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

``` 
Note: If using role="button" instead of the semantic <button> or <input type="button"> elements, you will need to make the element focusable and have to define event handlers for click and keydown events, including the Enter and Space keys, in order to process the user's input. See the official WAI-ARIA example code.
```

With a button everything works as intended, and from what I could see, even style. What do you think ?

**Comment by stefanorigano** [comment]
Yes, with `<button>` is visually ok.
I initially replaced it with `<span>` to handle design inconsistencies, but seems like it's not needed anymore following the latest updates on master. For me it's ok 👍 

@brboi , any feedback?

Out of scope of this task I guess, but I noticed that it's still not possible to select/edit the single filters using the keyboard (custom filters menu)

**Comment by kebeclibre** [comment]
> Out of scope of this task I guess, but I noticed that it's still not possible to select/edit the single filters using the keyboard (custom filters menu)
I tried in 14.0 and there is the same bug. 

Alright then, the customfilter menu is back to being a button. I put this pr up for merge. thanks @stefanorigano

**Comment by ged-odoo** [comment]
robodoo r+

**Pull request closed without merging**